frontend/store/reducers/userReducer.js

```diff
@@ -72,8 +68,6 @@ export const userLogin = async (response, dispatch) => {
     if (user && user.token) {
```

Comment:
I'm pretty sure this check for `user.token` needs to be removed here. I'll need to test this locally later to confirm.